examples/yaos_millionaires/README.md

```diff
@@ -13,14 +41,82 @@ $ cenclave localtest --code src/ \
                      --test tests/
 ```
 
-## Create Cosmian Enclave package with the code and the container image
+### Create a package for Cosmian Enclave
```

Comment:
A `package` is something specific to Cosmian or just a tarball ?

Reply:
Fixed in 966f5cb8099acb55b7881976b7964fb006dcf8ec.